### ISSUE TITLE
Add button 'repair packages'

### ISSUE THREAD

**Issue by dizzzz** [eXist-db/dashboard#16]
As described in the documentation, a button executing

``` xquery

xquery version "3.0";
import module namespace repair="http://exist-db.org/xquery/repo/repair" 
at "resource:org/exist/xquery/modules/expathrepo/repair.xql";
repair:clean-all(),
repair:repair()
```

would be very handy

**Comment by joewiz** [comment]
@dizzzz Would you still find this useful in dashboard 2.x?

**Comment by JoernT** [comment]
i agree with @dizzzz - this would be a handy function. However where to place it?

**Comment by joewiz** [comment]
Perhaps in Package Manager. But if packages need fixing, would Dashboard even be available? If not, then perhaps in the eXist Menu Bar / Task Menu or the Tool Window thereof or the Java Admin Client.

**Comment by dizzzz** [comment]
indeed, in the most ugly but realistic situation, the package manager is not there....

**Comment by dizzzz** [comment]
maybe in `/db/system/tools` or so ? or 'support' or 'scripts' or....
